Tasks/AzureResourceGroupDeploymentV2/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -27,6 +27,8 @@
   "loc.input.help.csmParametersFile": "Geben Sie den Pfad für die Parameterdatei für die Azure Resource Manager-Vorlage an oder ein Muster, das darauf verweist.",
   "loc.input.label.overrideParameters": "Vorlagenparameter überschreiben",
   "loc.input.help.overrideParameters": "Wenn Sie die Vorlagenparameter in einem Raster anzeigen möchten, klicken Sie neben dem Textfeld \"Parameter außer Kraft setzen\" auf \"...\". Für dieses Feature müssen CORS-Regeln an der Quelle aktiviert sein. Wenn sich Vorlagen im Azure Storage Blob befinden, finden Sie [hier](https://docs.microsoft.com/de-de/rest/api/storageservices/fileservices/Cross-Origin-Resource-Sharing--CORS--Support-for-the-Azure-Storage-Services?redirectedfrom=MSDN#understanding-cors-requests) Informationen zum Aktivieren von CORS. Sie können alternativ auch die zu überschreibenden Vorlagenparameter in das Textfeld eingeben. Beispiel: <br>–storageName fabrikam –adminUsername $(vmusername) -adminPassword $(password) –azureKeyVaultName $(fabrikamFibre).<br>Wenn der verwendete Parameterwert mehrere Wörter umfasst, schließen Sie ihn in Anführungszeichen ein – selbst dann, wenn Sie der Wert mithilfe von Variablen übergeben wird. Beispiel: -name \"parameter value\" -name2 \"$(var)\"<br>Verwenden Sie zum Außerkraftsetzen von Objekttypparametern als Zeichenfolgen dargestellte JSON-Objekte. Beispiel: -options [\"option1\"] -map {\"key1\": \"value1\" }.",
+  "loc.input.label.deploymentName": "Bereitstellungsname",
```

Comment:
You only should update the en-us resjson file. Other languages will be auto-generated. Please revert them.